docs/DeveloperGuide.md

```diff
@@ -332,16 +332,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `MediSync`, the **Actor** is the `user`, and the `precondition` is that user is in the main interface unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01- Add a person**
```

Comment:
Is it okay if you separate the UC tags from the dashes before you merge? i.e. `UC01 -`
Likewise for the other UC tags.

Will approve this so you can merge immediately after.